### PR TITLE
Member changes

### PR DESCRIPTION
**Pull request by davk28**

### CODE REVIEW

File: members/index.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ comments: false
   * IGN: <b>dakid80</b>
 * Stephen
   * IGN: <b>Way2Saucey</b>
+* Haru
```
Comment (drn) [inline members/index.md]: want to add ex-VGM here?

Comment (davk28) [inline members/index.md]: yeah

File: members/index.md [hunk 2]
```diff
@@ -49,18 +51,22 @@ comments: false
...
+* Rodrigo
+  * IGN: <b>xSharpScrolls</b>
+* Alejandro
+  * IGN: <b>MokaBlanco</b>
+* Alexander
+  * IGN: <b>AlexGG93</b>
+* Derek (currently in <b>Revive</b> guild)
   * IGN: <b>Haiwirbelsturm</b>
-* Alan (<b class='officer'>ex-VGM</b> in <b>S W A G</b> guild)
+* Alan (<b class='officer'>ex-VGM</b> now in <b>S W A G</b> guild)
```
Comment (drn) [inline members/index.md]: 👍

**Comment by drn** [comment]
make that change, squash it into a single commit and then we can get it merged in!

**Pull request merged**